index.d.ts

```diff
@@ -208,6 +264,21 @@ declare namespace Emittery {
 		// `[eventName: EventName]: unknown;`
 	}
 
+	/**
+	The data provided as `eventData` when listening for `Emittery.listenerAdded` or `Emittery.listenerRemoved`.
+	*/
+	interface ListenerChangedData {
+		/**
+		The listener that was added or removed.
+		*/
+		listener: (eventData?: unknown) => void;
+
+		/**
+		The name of the event that was added or removed if `.on()` or `.off()` was used, or `undefined` if `.onAny()` or `.offAny()` was used.
+		*/
+		eventName?: EventName;
```

Comment:
I just realized that we don't add the `on(eventName: typeof Emittery.listenerAdded | typeof Emittery.listenerRemoved` overload to `onAny`/`offAny`, so they can never receive this TS interface, so `eventName` doesn't need to be optional.

Reply:
@sindresorhus We are responding to `.onAny`, not using it directly.

```js
const Emittery = require(".")

const emitter = new Emittery()

emitter.on(Emittery.listenerAdded, ({listener, eventName}) => {
	console.log(listener)
	//=> [Function (anonymous)]

	console.log(eventName)
	//=> undefined
})

emitter.onAny(() => {
	// Process logic...
})
```

Reply:
Right, never mind. I was confused.